tests/integrations/android_sdk/AndroidSDK/test__create_emulator.py

```diff
@@ -184,6 +183,69 @@ def test_create_emulator_with_defaults(
     assert "skin.name=pixel_7_pro" in config
 
 
+def test_create_emulator_with_nonexistent_avd_home(
```

Comment:
I spent way too long trying to add an ability to check if `mkdir()` was called in the existing tests...and nothing was working....so, there's this test too now.